uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -55,12 +56,12 @@
 @Table(name = JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX)
 @SequenceGenerator(
         name=JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX + "_GEN",
-        sequenceName="UP_PAGS_TEST_SEQ",
+        sequenceName=JpaPersonAttributesGroupTestDefinitionDao.TABLENAME_PREFIX + "_SEQ",
```

Comment:
Looks like this PR has my changes in it as well.  We can re-roll it with some Git magic to take out this "noise."